src/components/PasswordReset/index.jsx

```diff
@@ -0,0 +1,93 @@
+import { Alert, Input } from "antd";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import { useTranslation } from "react-i18next";
+
+const PasswordReset = (props) => {
+  const [t] = useTranslation();
+  const [passwordResetInfo, setPasswordResetInfo] = useState({ email: "" });
+  const handleChange = (key, value) => {
+    let newValues = Object.assign({}, passwordResetInfo);
+    newValues[key] = value;
+    setPasswordResetInfo(newValues);
+  };
+  const [errors, setErrors] = useState({});
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let { email } = passwordResetInfo;
+    let errors = {};
+    if (!validateEmail(email)) {
+      errors["email"] = t("passwordReset.emailFormat");
+    }
+    if (email.trim() === "") {
```

Comment:
shouldn't this just be part of validate email. I bet of you console logged line 23 you can't make it print, because the previous one would have caught it

Reply:
condition deleted PTAL